[2018-09-03 17:15:31,228] {cli.py:374} INFO - Running on host MacBook-Pro-de-MANGANNEAU.local
[2018-09-03 17:15:31,244] {models.py:1197} INFO - Dependencies all met for <TaskInstance: workshop_airflow_exo_1.parse_json 2018-09-03 15:15:16.621125 [queued]>
[2018-09-03 17:15:31,250] {models.py:1197} INFO - Dependencies all met for <TaskInstance: workshop_airflow_exo_1.parse_json 2018-09-03 15:15:16.621125 [queued]>
[2018-09-03 17:15:31,250] {models.py:1407} INFO - 
--------------------------------------------------------------------------------
Starting attempt 1 of 1
--------------------------------------------------------------------------------

[2018-09-03 17:15:31,256] {models.py:1428} INFO - Executing <Task(PythonOperator): parse_json> on 2018-09-03 15:15:16.621125
[2018-09-03 17:15:31,256] {base_task_runner.py:115} INFO - Running: ['bash', '-c', 'airflow run workshop_airflow_exo_1 parse_json 2018-09-03T15:15:16.621125 --job_id 171 --raw -sd DAGS_FOLDER/exo1.py']
[2018-09-03 17:15:32,095] {base_task_runner.py:98} INFO - Subtask: [2018-09-03 17:15:32,094] {__init__.py:45} INFO - Using executor SequentialExecutor
[2018-09-03 17:15:32,141] {base_task_runner.py:98} INFO - Subtask: [2018-09-03 17:15:32,141] {models.py:189} INFO - Filling up the DagBag from /Users/magus/PycharmProjects/workshop_airflow/dags/exo1.py
[2018-09-03 17:15:32,227] {cli.py:374} INFO - Running on host MacBook-Pro-de-MANGANNEAU.local
[2018-09-03 17:15:32,245] {models.py:1595} ERROR - 'list' object cannot be interpreted as an integer
Traceback (most recent call last):
  File "/Users/magus/PycharmProjects/workshop_airflow/workshop_env/lib/python3.6/site-packages/airflow/models.py", line 1493, in _run_raw_task
    result = task_copy.execute(context=context)
  File "/Users/magus/PycharmProjects/workshop_airflow/workshop_env/lib/python3.6/site-packages/airflow/operators/python_operator.py", line 89, in execute
    return_value = self.execute_callable()
  File "/Users/magus/PycharmProjects/workshop_airflow/workshop_env/lib/python3.6/site-packages/airflow/operators/python_operator.py", line 94, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
  File "/Users/magus/PycharmProjects/workshop_airflow/utils/parsing.py", line 39, in parse_journey
    for j in range(0, number_of_section):
TypeError: 'list' object cannot be interpreted as an integer
[2018-09-03 17:15:32,248] {models.py:1624} INFO - Marking task as FAILED.
[2018-09-03 17:15:32,254] {models.py:1644} ERROR - 'list' object cannot be interpreted as an integer
[2018-09-03 17:15:32,257] {base_task_runner.py:98} INFO - Subtask: Traceback (most recent call last):
[2018-09-03 17:15:32,257] {base_task_runner.py:98} INFO - Subtask:   File "/Users/magus/PycharmProjects/workshop_airflow/workshop_env/bin/airflow", line 27, in <module>
[2018-09-03 17:15:32,257] {base_task_runner.py:98} INFO - Subtask:     args.func(args)
[2018-09-03 17:15:32,257] {base_task_runner.py:98} INFO - Subtask:   File "/Users/magus/PycharmProjects/workshop_airflow/workshop_env/lib/python3.6/site-packages/airflow/bin/cli.py", line 392, in run
[2018-09-03 17:15:32,257] {base_task_runner.py:98} INFO - Subtask:     pool=args.pool,
[2018-09-03 17:15:32,257] {base_task_runner.py:98} INFO - Subtask:   File "/Users/magus/PycharmProjects/workshop_airflow/workshop_env/lib/python3.6/site-packages/airflow/utils/db.py", line 50, in wrapper
[2018-09-03 17:15:32,257] {base_task_runner.py:98} INFO - Subtask:     result = func(*args, **kwargs)
[2018-09-03 17:15:32,257] {base_task_runner.py:98} INFO - Subtask:   File "/Users/magus/PycharmProjects/workshop_airflow/workshop_env/lib/python3.6/site-packages/airflow/models.py", line 1493, in _run_raw_task
[2018-09-03 17:15:32,280] {base_task_runner.py:98} INFO - Subtask:     result = task_copy.execute(context=context)
[2018-09-03 17:15:32,281] {base_task_runner.py:98} INFO - Subtask:   File "/Users/magus/PycharmProjects/workshop_airflow/workshop_env/lib/python3.6/site-packages/airflow/operators/python_operator.py", line 89, in execute
[2018-09-03 17:15:32,281] {base_task_runner.py:98} INFO - Subtask:     return_value = self.execute_callable()
[2018-09-03 17:15:32,281] {base_task_runner.py:98} INFO - Subtask:   File "/Users/magus/PycharmProjects/workshop_airflow/workshop_env/lib/python3.6/site-packages/airflow/operators/python_operator.py", line 94, in execute_callable
[2018-09-03 17:15:32,281] {base_task_runner.py:98} INFO - Subtask:     return self.python_callable(*self.op_args, **self.op_kwargs)
[2018-09-03 17:15:32,281] {base_task_runner.py:98} INFO - Subtask:   File "/Users/magus/PycharmProjects/workshop_airflow/utils/parsing.py", line 39, in parse_journey
[2018-09-03 17:15:32,281] {base_task_runner.py:98} INFO - Subtask:     for j in range(0, number_of_section):
[2018-09-03 17:15:32,282] {base_task_runner.py:98} INFO - Subtask: TypeError: 'list' object cannot be interpreted as an integer
